packages/contracts/src/abis/index.js

```diff
@@ -0,0 +1,17 @@
+import IBEX from "./ibex"
+import LYNX from "./lynx"
+import MAINNET from "./mainnet"
+
+
+const abis = {
+    1:      MAINNET,
+    "0x1":  MAINNET,
+
+    4:      IBEX,
+    "0x4":  IBEX,
+
+    "0x5":  LYNX,
+    5:      LYNX,
```

Comment:
What's necessitating the double entries here?

Reply:
Because the provider.chainID is in hex... and it's annoying to have to always parseInt() on it.
But I felt like there would be other times when we'd want to get it by integer... so it's for convenience.